dashboard/USER_GUIDE.md

```diff
@@ -13,36 +13,23 @@ at least one test has failed against the top of tree and flutter/flutter is
 currently not accepting new commits. Check on Discord [#tree-status](https://discord.com/channels/608014603317936148/613398423093116959)
 for discussion about tree closures.
 
-Tree closures occur when the latests results for a test have returned as
-failing. Logs are found by clicking a task box then clicking "Download Logs."
+Tree closures occur when the latest results for a test have returned as
+failing. Logs are found by clicking a task box then clicking "view logs."
 
-### Why am I not able to download logs?
-
-Ensure that you are signed in to the app (top right).
-
-Reach out on Discord [#hackers-infra](https://discord.com/channels/608014603317936148/608021351567065092)
-to be added as an AllowListedAccount to Cocoon.
-
-When the DeviceLab migrates to LUCI, logs will be available without sign in.
 
 ### Why is a task stuck on "new task" status?
 
 The dashboard aggregates build results from multiple build environments,
-including Cirrus, LUCI, and DeviceLab. DeviceLab and Cirrus test every commit
-that goes into the `master` branch. However, LUCI may skip commits when they
-come in too fast.
+including Cirrus, LUCI, and DeviceLab. Due to limited capacity to run tests on
+physical devices, some tests may be batched (run for several commits).
 
 Flutter infra prioritizes running tasks against the most recent commits. This
 leads to some tasks never being run on a commit as the test coverage was
 provided from a more recent commit.
 
-DeviceLab tests are eventually run against every commit to ensure benchmark
-data is collected from every commit. This helps with triaging where performance
-regressions started.
-
 ### Why are some tasks outlined instead of solid?
 
-Outlined tasks indicate a task that is running experimentally. This can be for
+Outlined tasks indicate a task that is running as bringup. This can be for
```

Comment:
Nit: a quick link to bringup.

Reply:
I removed this as there's info inside the legend of the app. Additionally, I linked to the general ci yaml docs explaining that it generates this dashboard